src/license_manager_agent_ops.py

```diff
@@ -231,3 +231,16 @@ def remove_license_manager_agent(self):
             self._ETC_DEFAULT.unlink()
         rmtree(self._LOG_DIR.as_posix(), ignore_errors=True)
         rmtree(self._VENV_DIR.as_posix(), ignore_errors=True)
+
+    @property
+    def fluentbit_config_lm_log(self) -> list:
+        """Return Fluentbit configuration parameters to forward LM agent logs."""
+        cfg = [{"input": [("name",             "tail"),
+                          ("path",             "/var/log/license-manager-agent/*.log"),
+                          ("multiline.parser", "multiline-lm")]},
```

Comment:
```suggestion
        cfg = [{"input": [("name",             "tail"),
                          ("path",             "/var/log/license-manager-agent/*.log"),
                          ("tag",             "lm.*"),
                          ("multiline.parser", "multiline-lm")]},
```

The tag is import for routability.

Reply:
I added it in the latest commit.